eng/Versions.props

```diff
@@ -36,7 +36,7 @@
     manually enabled by updating the metadata.
   -->
   <ItemGroup>
-    <ProjectServicingConfiguration Include="Microsoft.WindowsDesktop.App.Ref" PatchVersion="0" />
+    <ProjectServicingConfiguration Include="Microsoft.WindowsDesktop.App.Ref" PatchVersion="6.0.0" />
```

Comment:
This is wrong. PatchVersion should not be the full version but just the last digit. AFAIK this setting is only relevant during servicing. @jkoritzinsky and @dagood should know more about it.

Reply:
Viktor is right here.

Reply:
Looks like including `("major.minor.patch".)` in the comment above this group might be confusing. Intended it to just be an example of where patch version is in a full version number, but easy to interpret differently. Probably clearer just to remove it and let the rest of the comment clarify that it should normally be the same as `PatchVersion`.